src/d3a/d3a_core/cli.py

```diff
@@ -71,9 +71,6 @@ def main(log_level):
 @click.option('-c', '--cloud-coverage', type=int,
               default=ConstSettings.PVSettings.DEFAULT_POWER_PROFILE, show_default=True,
               help="Cloud coverage, 0 for sunny, 1 for partial coverage, 2 for clouds.")
-@click.option('-f', '--iaa-fee', type=int,
```

Comment:
Just to make sure: We decided in the offline discussion, that we remove this cli parameter but keep it in the SimulationConfig class for the d3a-web to be set, right? (need to know for my current story)